src/ResourceManager/Network/Commands.Network/AzureRM.Network.psd1

```diff
@@ -214,6 +214,8 @@ CmdletsToExport = 'Add-AzureRmApplicationGatewayAuthenticationCertificate',
                'Test-AzureRmPrivateIPAddressAvailability', 
                'Get-AzureRmPublicIpAddress', 'New-AzureRmPublicIpAddress', 
                'Remove-AzureRmPublicIpAddress', 'Set-AzureRmPublicIpAddress', 
+               'Get-AzureRmPublicIpPrefix', 'New-AzureRmPublicIpPrefix', 
```

Comment:
you will need to update the [netcore psd1](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Network/Commands.Network/AzureRM.Network.Netcore.psd1) as well

Reply:
Done